service/controller/v18/cloudconfig/worker_template.go

```diff
@@ -3,8 +3,9 @@ package cloudconfig
 import (
 	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/certs"
-	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_3_7_3"
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_4_0_0"
```

Comment:
Do we plan to rename the repo at some point? What is the idea here? /cc @giantswarm/sig-operator

Reply:
I think so. We can do that together with VOO when we won't have to use version packages.